lib/constants/enums.ts

```diff
@@ -65,10 +65,15 @@ export enum ReputationEvent {
   ManualBan = 0,
   ManualUnban = 1,
   PacketTimeout = 2,
+  /** When a swap is accepted and is attempted to be executed but fails. */
   SwapFailure = 3,
   SwapSuccess = 4,
   WireProtocolErr = 5,
   InvalidAuth = 6,
+  /** When a swap is accepted and then fails due to exceeding time limits. */
+  SwapTimeout = 7,
+  /** When a swap fails due to unexpected or possibly malicious behavior. */
+  SwapMisbehavior = 8,
```

Comment:
Perhaps group the swap types togeather (I think it's still fine to do breaking changes), and explain a bit the difference between `SwapFailure` and `SwapTimeout`/`SwapMisbehavior`, since they also indicate a failure.

Reply:
I grouped them together, but I'm not sure exactly what more to say about the differences between the errors without getting too verbose or specific. I tried outlining the general differences with the existing comments, let me know if you have anything in mind.